app/models/lockstep/user.rb

```diff
@@ -7,4 +7,27 @@ class Lockstep::User < Lockstep::ApiRecord
   belongs_to :account, class_name: "Lockstep::Account", loader: ->(user) { Lockstep::Account.first }
 
   alias_attribute :last_logged_in, :b2_c_last_logged_in
+
+  def self.invite(email_addresses)
+    emails = email_addresses.map { |email| { email: email } }
+    resp = resource.post('/invite', body: emails, params: {})
+    parsed_response = JSON.parse(resp.body)
+    if resp.code != '200'
+      if resp.code == '401'
+        raise Lockstep::Exceptions::UnauthorizedError, parsed_response['title']
+      elsif resp.code == '400'
+        raise Lockstep::Exceptions::BadRequestError, parsed_response['title']
+      elsif resp.code == '404'
+        raise Lockstep::Exceptions::RecordNotFound, parsed_response['title']
+      end
+    end
+    result = parsed_response&.map do |user|
+      if user['success']
+        user['invitedUser'] = Lockstep::User.find_by(user_id: user['invitedUser']['userId'])
+        user
+      elsif !user['success']
+        user
```

Comment:
If its not a success, lets initialise the `Platform::User` object and set it as errors using `record.errors.add`. We should keep this consistent with rails.

Reply:
if it is not a success message it can be that case as well where the user is already invited message comes form the platform. So that is for that and we are not changing that whatever object comes from platform for those cases we are keeping and sending those.